account_fiscal_position_rule_stock/stock.py

```diff
@@ -75,3 +75,109 @@ def _prepare_invoice(self, cr, uid, picking, partner, inv_type,
         result['fiscal_position'] = picking.fiscal_position and \
             picking.fiscal_position.id
         return result
+
+
+class stock_picking_out(osv.Model):
+    _inherit = "stock.picking.out"
+    _columns = {
+        'fiscal_position': fields.many2one(
+            'account.fiscal.position', 'Fiscal Position',
+            domain="[('fiscal_operation_id','=',fiscal_operation_id)]"),
```

Comment:
hehe this explains why! Ok fine then. For a minute we got some excitement to believe some core or OCA object would bring our localization closer. But OK, anyway we that kind of code coming into fiscal rules we are converging so please clean up and we evaluate again. We are pushing great stuff for v8, sorry for the delay but it's coming.